### PR TITLE
A bit of 💅

### PR DESCRIPTION
**Pull request by folkyatina**

### CODE REVIEW

File: contracts/0.8.9/test_helpers/NodeOperatorsRegistryMockForLidoOracleNew.sol [hunk 1]
```diff
@@ -1,5 +1,4 @@
-// SPDX-FileCopyrightText: 2021 Lido <info@lido.fi>
```
Comment (skozin) [inline contracts/0.8.9/test_helpers/NodeOperatorsRegistryMockForLidoOracleNew.sol:1]: This is not used anywhere

File: contracts/0.8.9/test_helpers/NodeOperatorsRegistryMockForSecurityModule.sol [hunk 2]
```diff
@@ -1,5 +1,4 @@
-// SPDX-FileCopyrightText: 2021 Lido <info@lido.fi>
-
+// SPDX-FileCopyrightText: 2023 Lido <info@lido.fi>
```
Comment (skozin) [inline contracts/0.8.9/test_helpers/NodeOperatorsRegistryMockForSecurityModule.sol:1]: This is not used anywhere as well?

File: docs/oracle-operator-manual.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ The daemon also fetches historical stETH token price (shifted by fifteen blocks)
...
 
-Communication between Ethereum 1.0 part of the system and the Beacon network is performed by the DAO-assigned oracles. They monitor staking providers’ Beacon chain accounts and submit corresponding data to the `LidoOracle` contract. The latter takes care of making sure that quorum about the data being pushed is reached within the oracles and enforcing data submission order (so that oracle contract never pushes data that is older than the already pushed one).
+Communication between Ethereum 1.0 part of the system and the Beacon network is performed by the DAO-assigned oracles. They monitor staking providers’ Beacon chain accounts and submit corresponding data to the `LegacyOracle` contract. The latter takes care of making sure that quorum about the data being pushed is reached within the oracles and enforcing data submission order (so that oracle contract never pushes data that is older than the already pushed one).
```
Comment (skozin) [inline docs/oracle-operator-manual.md:22]: This is not correct

File: docs/oracle-operator-manual.md [hunk 4]
```diff
@@ -19,9 +19,9 @@ The daemon also fetches historical stETH token price (shifted by fifteen blocks)
...
 
-Upon every update submitted by the `LidoOracle` contract, the system recalculates the total stETH token balance. If the overall staking rewards are bigger than the slashing penalties, the system registers profit, and fee is taken from the profit and distributed between the insurance fund, the treasury, and node operators.
+Upon every update submitted by the `LegacyOracle` contract, the system recalculates the total stETH token balance. If the overall staking rewards are bigger than the slashing penalties, the system registers profit, and fee is taken from the profit and distributed between the insurance fund, the treasury, and node operators.
```
Comment (skozin) [inline docs/oracle-operator-manual.md:24]: and this

File: docs/oracle-operator-manual.md [hunk 5]
```diff
@@ -38,15 +38,15 @@ In order to launch oracle daemon on your machine, you need to have several thing
 
 ## The oracle daemon
 
-The oracle daemon is a simple Python app that watches the Beacon chain and pushes the data to the LidoOracle Smart Contract: [Mainnet](https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb) / [Görli](https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F).
+The oracle daemon is a simple Python app that watches the Beacon chain and pushes the data to the LegacyOracle Smart Contract: [Mainnet](https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb) / [Görli](https://goerli.etherscan.io/address/0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F).
```
Comment (skozin) [inline docs/oracle-operator-manual.md:41]: this is not correct

Comment (folkyatina) [reply docs/oracle-operator-manual.md:41]: the whole document is not correct as I understand

File: docs/protocol-levers.md [hunk 6]
```diff
@@ -173,7 +173,7 @@ Allow to manage signing keys for the given node operator.
 Allows to report that `_stoppedIncrement` more validators of a node operator have become stopped.
 
 
-## [LidoOracle.sol](/contracts/0.4.24/oracle/LidoOracle.sol)
+## [LegacyOracle.sol](/contracts/0.4.24/oracle/LegacyOracle.sol)
```
[the remark this answers is not in the document]
Comment (folkyatina) [reply docs/protocol-levers.md:176]: I prefer to delete it. I think it's completely wrong right now and will be replaced by spec in near future

**Pull request merged**